browser/net/brave_static_redirect_network_delegate_helper.cc

```diff
@@ -1,9 +1,10 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
```

Comment:
i think the MPL header is still needed. at least in the reviewer checklist it says `New files have MPL-2.0 license header.`

Reply:
cc: @bridiver - 

Should we remove the `New files have MPL-2.0 license header.` item from the reviewer checklist. Or use this header:

```
/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
```